src/index.js

```diff
@@ -76,8 +81,9 @@ export default function supernova(env) {
         if (layout && tableData && !env.carbon) {
           registerLocale(translator);
           const changeSortOrder = sortingFactory(model);
-          render(rootElement, {
+          render({
             rootElement,
+            reactRoot,
```

Comment:
Don't put it in this object, no need to pass it to the rest of the app

Reply:
Fixed